appveyor.yml

```diff
@@ -1,5 +1,15 @@
 version: 1.0.{build}
 
+install:
+  - git submodule update --init --recursive  
+
 build_script:
-  - git submodule update --init --recursive
   - vcbuild.bat MSVC2013
+  
+before_test:
+  - gem install cucumber
```

Comment:
Yes, but this is an improvement which we are trying to merge into shared/mson branch. We don't need to back port this to master branch.